lms/templates/ccx/coach_dashboard.html

```diff
@@ -35,11 +35,16 @@
           </ul>
         % endif
         <section>
-          <form action="${create_ccx_url}" method="POST">
+          <p class="request-response-error" id="ccx-create-message"></p>
+          <form action="${create_ccx_url}" class="ccx-form" method="POST" onsubmit="return validateForm(this)">
             <input type="hidden" name="csrfmiddlewaretoken" value="${csrf_token}"/>
-            <label class="sr" for="ccx_name">${_('Name your CCX')}</label>
-            <input name="name" id="ccx_name" placeholder="${_('Name your CCX')}"/><br/>
-            <button id="create-ccx">Coach a new Custom Course for EdX</button>
+            <div class="field">
+              <label class="sr" for="ccx_name">${_('Name your CCX')}</label>
+              <input name="name" id="ccx_name" placeholder="${_('Name your CCX')}"/><br/>
+            </div>
+            <div class="field">
+              <button id="create-ccx" type="submit">Coach a new Custom Course for EdX</button>
```

Comment:
Change button to `Create a new Custom Course for edX` (not capitalization of edX). 

It needs to be wrapped for translation also.